btcpay-setup.sh

```diff
@@ -1,22 +1,49 @@
 #!/bin/bash
 
-if [ "$0" = "$BASH_SOURCE" ]; then
+set +x
+
+if [[ "$0" = "$BASH_SOURCE" ]]; then
     echo "This script must be sourced \". btcpay-setup.sh\"" 
     exit 1
 fi
 
-if [[ $EUID -ne 0 ]]; then
-   echo "This script must be run as root after running \"sudo su -\"" 
-   return
+if [[ "$OSTYPE" == "darwin"* ]]; then
+	# Mac OS
+
+	if [[ $EUID -eq 0 ]]; then
+		# Running as root is discouraged on Mac OS. Run under the current user instead.
+        echo "This script should not be run as root."
+        return
+    fi
+
+	BASH_PROFILE_SCRIPT="$HOME/btcpay-env.sh"
+
+	# Mac OS doesn't use /etc/profile.d/xxx.sh. Instead we create a new file and load that from ~/.bash_profile
+	if [ ! -f "$FILE" ]; then
```

Comment:
what is `$FILE`?

Reply:
Woops. Something must've gone wrong with stashing my changes and moving files back and forth between my test VM. 

Fixed it